test/verify/check-login

```diff
@@ -162,8 +162,11 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
 
             # login with user shell that prints some stdout/err noise
             # having stdout output in ~/.bashrc confuses docker, so don't run on Atomic
-            m.execute("echo 'echo noise-rc-out; echo noise-rc-err >&2' >> ~admin/.bashrc")
-            m.execute("echo 'echo noise-profile-out; echo noise-profile-err >&2' >> ~admin/.profile")
+            m.execute("set -e; cd ~admin; cp -a .bashrc .bashrc.bak; [ ! -e .profile ] || cp -a .profile .profile.bak; "
```

Comment:
Wondering why you do `cd ~admin` and then use `~admin/.bashrc`. Either no `cd` is needed, or then you can just point to `.bashrc`.

Reply:
Good point, thanks! I just forgot to clean this up. Pushed.